Main.java

```diff
@@ -9,7 +10,16 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+   Scanner input = new Scanner(System.in);
+   // Ask for integers
+   System.out.println("Please enter in two integers, on seperate lines, to divide.:");
+   // declaring numerator and denominator variables
+    int numerator = input.nextInt();
+    int denominator = input.nextInt();
+    // declaring quotient and remainder variables
+    int quotient = numerator/denominator;
+    int remainder = numerator%denominator;
+    // stating the answer
+    System.out.println(quotient + " with a of remainder " + remainder);

```

Comment:
Your variable names, comments, and math are fantastic. However, your output is a much different format than shown. Please try and match this exactly.